modules/akkaHttp/src/main/scala/tschema/akkaHttp/FindKey.scala

```diff
@@ -1,5 +1,5 @@
-package ru.tinkoff.tschema.akkaHttp
-import ru.tinkoff.tschema.common.Find
+package tschema.akkaHttp
```

Comment:
We have to add deprecated aliases for old names.

Reply:
Maybe just write a scalafix rule?